apps/src/studio/levels.js

```diff
@@ -2240,10 +2292,10 @@ levels.js_hoc2015_add_characters = {
     }
   ],
 
-  'requiredForSuccess' : {
-    'removedItemCount': 3,
-    'winGame': true
-  }
+  'progressConditions' : [
+    { required: { 'timedOut': true, 'collectedItemsBelow': 3 }, 
+      result: { 'success': false, message: msg.failedAddCharactersTimeout() } }
+  ]
```

Comment:
This one looks incomplete.  Does the level have to time out before we check for success?